docs/contributing.rst

```diff
@@ -32,7 +32,7 @@ the team and contributors are working on, you can join our public team calls:
 
 - Mondays and Wednesdays at 3pm CET/CEST.
```

Comment:
```suggestion
- Mondays and Wednesdays at 3PM CET/CEST.
```

Reply:
Correct, it does help with readability, especially on mobile devices. It's part of some UX recommendations for speed reading.